data/actionsmap/yunohost.yml

```diff
@@ -436,7 +436,7 @@ app:
                     help: Custom name for the app
                 -a:
                     full: --args
-                    help: Serialize arguments for app installation
+                    help: Serialized arguments for app script (i.e. "domain=domain.tld&url_path=/path")
```

Comment:
But "domain" and "path" are kind of reserved variable names, and are automatically saved on app_install ( https://github.com/yunohost/yunohost/blob/unstable/src%2Fyunohost%2Fapp.py#L876 ).
Using `url_path` could lead to some variable name mixin as in https://github.com/M5oul/sonerezh_ynh/blob/master/scripts%2Fupgrade#L9
But your app, your choice ;)